src/controllers/userController.ts

```diff
@@ -1,112 +1,79 @@
 import { Request, Response } from "express";
-import { ICreateUser } from "../helpers/dto";
+import { StatusCodes } from "http-status-codes";
+import { IUser } from "../helpers/dto";
 import { userService } from "../services/userService";
 const userServiceInstance = new userService();
 
 export class userController {
   async createUser(request: Request, response: Response): Promise<Response> {
     try {
-      const {
-        username,
-        firstname,
-        lastname,
-        email,
-        password,
-        position,
-        age,
-        gender,
-      }: ICreateUser = request.body;
       const user = await userServiceInstance.createUser({
-        username,
-        firstname,
-        lastname,
-        email,
-        password,
-        position,
-        age,
-        gender,
+        ...request.body as IUser
       });
-      return response.status(201).send(user);
+      return response.status(StatusCodes.OK).send(user);
```

Comment:
```suggestion
      return response.status(StatusCodes.Created).send(user);
```

Since we're creating a new user, should be `StatusCodes.Created`